fabtests/component/dmabuf-rdma/fi-rdmabw-xe.c

```diff
@@ -1137,11 +1136,24 @@ int main(int argc, char *argv[])
 	char *s;
 	int loc1 = MALLOC, loc2 = MALLOC;
 
-	while ((c = getopt(argc, argv, "2d:D:e:p:m:n:t:gPB:rRsS:x:hv")) != -1) {
+	while ((c = getopt(argc, argv, "2b:d:D:e:p:m:n:t:gPB:rRsS:x:hv")) != -1) {
 		switch (c) {
 		case '2':
 			bidir = 1;
 			break;
+		case 'b':
+			batch = atoi(optarg);
+			if (batch <= 0) {
+				fprintf(stderr,
+					"Batch too small, adjusted to 1\n");
+				batch = 1;
+			} else if (batch > TX_DEPTH) {
+				fprintf(stderr,
+					"Batch too large, adjusted to %d\n",
+					TX_DEPTH);
+				batch = TX_DEPTH;
+			}
```

Comment:
Does this option serve any significant purpose for testing?

Reply:
Main purpose is to see how completions affect performance.